lib/BloqadeQMC/src/ising/Rydberg.jl

```diff
@@ -69,86 +69,37 @@ function make_prob_vector(H::Type{<:AbstractRydberg}, V::UpperTriangular{T}, Ω:
     return ops, p, energy_shift
 end
 
+total_hx(H::Rydberg)::Float64 = sum(H.Ω) / 2
+haslongitudinalfield(H::AbstractRydberg) = !iszero(H.δ)
 
-###############################################################################
-
-# function BlockadeRydberg(dims::NTuple{N, Int}, J::Float64, hx::Float64, hz::Float64, pbc=true) where N
-# end
-
-function Rydberg(dims::NTuple{D, Int}, R_b, Ω, δ; pbc=true, trunc::Int=0, epsilon::Float64=0) where D
-    if D == 1
-        lat = Chain(dims[1], 1.0, pbc)
-    elseif D == 2
-        lat = Rectangle(dims[1], dims[2], 1.0, 1.0, pbc)
-    else
-        error("Unsupported number of dimensions. 1- and 2-dimensional lattices are supported.")
-    end
-    return Rydberg(lat, R_b, Ω, δ; trunc=trunc, epsilon=epsilon)
+function _make_vector(param::AbstractVector, Ns::Int)
+    return param
 end
 
+function _make_vector(param::Real, Ns::Int)
```

Comment:
why is this function even needed? isn't it just one-line broadcasting? If this is needed as an internal function, do not use `_` prefix, think about a better name explaining what it does than vaguely name if `make_vector`

Reply:
make_prob_vector() only takes vectors currently, so we can't rely on dispatch / broadcasting there.
But I'll take away the `_` prefix!

Reply:
I don't mean just remove `_` prefix, I'm saying this function name means nothing to a reader, this is not good practice, e.g what vector it makes? if you cannot figure out a proper name explain what it does, write a comment or just don't define it.